src/debian/12/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM debian:bookworm
+
+# Install Helix Dependencies
+
+RUN apt-get update && \
+    apt-get install -y \
+        autoconf \
+        automake \
+        at \
+        build-essential \
+        curl \
```

Comment:
I don't see cmake listed here. It's listed in the Arm64 version and also in the earlier Bullseye version.

Reply:
I don't think `cmake` its needed for Helix. (nor is gcc etc.. e.g. we generally don't need to build to run tests) 
We may make the images smaller if we don't intend them as build images.

Reply:
Are you sure, Matt. I copied the 11 version of the same thing. I think you are looking at the wrong Dockerfile. Make sure the paths match.

Reply:
Ah, sorry. I was looking at this: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/79dc2e51d29d9c755b00c216699160d096726d9c/src/debian/11/amd64/Dockerfile#L11

I didn't notice that your changes are for the Helix Dockerfile. Never mind. Carry on.

Reply:
Right. Easy to do.